pyrdp/ui/PlayPauseButton.py

```diff
@@ -32,4 +33,4 @@ def setPlaying(self, playing):
             self.setIcon(QIcon.fromTheme("media-playback-pause"))
         else:
             self.setText(self.playText)
-            self.setIcon(QIcon.fromTheme("media-playback-start"))
\ No newline at end of file
+            self.setIcon(QIcon.fromTheme("media-playback-start"))
```

Comment:
Try to avoid unrelated changes, use `git add -p` (or the GUI equivalent) instead of `git add` to select what you commit.

Not for this PR but for the next one

Reply:
Yeah, will do.